README.md

```diff
@@ -461,6 +461,73 @@ TL;DR for a simple backup of three PostgreSQL databases in the same project name
 NOTE the `BACKUP_VOLUME_NAME=bk-b7cg3n-deploy-yxq6rf8z23pu` is from Step 2 above; when using the GUI there is no option to set the PVC name.
 </details>
 
+<details><summary>Concrete example of a MongoDB deployment</summary>
+
+
+TL;DR: A simple backup of a single MongoDB database with backup validation.
+
+1. Decide on amount of backup storage required (5Gi is currently the maximum)
+2. Provision the nfs-backup PVC, following the [docs](https://github.com/BCDevOps/provision-nfs-apb/blob/master/docs/usage-gui.md). This provisioning may take several minutes to an hour, and if using the GUI, will result in a PVC with a name similar to `bk-abc123-dev-v9k7xgyvwdxm`, where `abc123-dev` is your project namespace and the last portion is randomly generated.
```

Comment:
Backup storage and provisioning of `nfs-backup` class storage is documented here; https://github.com/BCDevOps/backup-container#backup-storage-volume.  I'd recommend referring to that section of the documentation.  Any additional details should be added there.

Reply:
Once again, taken verbatim from the Postgres example. I'll use this opportunity to correct this in both instances.

Reply:
Perfect, thanks @ashtonmeuser!